### ISSUE TITLE
Display a specific error message when initPKSim() cannot find proper PKSIM version.

### ISSUE THREAD

**Issue by Felixmil** [Open-Systems-Pharmacology/OSPSuite-R#1269]
I encountered this error after updating OSPSuite-R:

```
Caused by error in `initPKSim()`:
! Could not find an installation of PK-Sim on the machine. Please install the OSPSuite or use 'initPKSim()' to specify the installation path
```

I suspected a problem with the installed version of PKSIM on my computer but could not be sure it was actually this issue. I fixed it after Pavel provided this [link ](https://ci.appveyor.com/project/open-systems-pharmacology-ci/pk-sim/branch/develop/artifacts) and explained what to do.

Is it possible to detect that the installed PKSIM is not the good version ? If yes, the error should explain that. Also, including a link to the latest release/dev version to get would be nice.

**Comment by Yuri05** [comment]
Yes, would be good to have more info in the error message. E.g. somethink like this:
```
PK-Sim version X.Y was not found on your system. To fix this:
- Either install the OSP Suite X.Y (s. https://github.com/Open-Systems-Pharmacology/Suite/releases)
- Or download and extract PK-Sim portable version X.Y (s. https://github.com/Open-Systems-Pharmacology/pk-sim/releases)

In the latter case, you need to call initPKSim() manually and provide the full path to the portable PK-Sim version (e.g. initPKSim(pksimFolderPath = "C:/PKSim_portable_11.2") ) 
```

**Comment by Felixmil** [comment]
@Yuri05 Would it be possible to distinguish between:
- PK-Sim is not installed at all
- PK-Sim version is not compatible

And have two error messages accordingly ?

**Comment by Yuri05** [comment]
Why would we need 2 different error messages? 
If the required PK-Sim version was not found, it does not really matter, if there is another version or no installation at all. Imho.
